napari/util/tests/test_keybindings.py

```diff
@@ -0,0 +1,230 @@
+import pytest
```

Comment:
Will these tests now get automatically run as part of our CI?

Reply:
Yes, any file starting with `test_` will have all `test_*` methods run automatically by `pytest`.

Reply:
oh great!